crates/telemetry/src/state.rs

```diff
@@ -381,6 +384,11 @@ impl TelemetryState {
                     "Number of errors observed by Hermes when broadcasting a Tx",
                 )
                 .init(),
+
+                filtered_packets: meter
+                .u64_counter("filtered_packets")
+                .with_description("Number of Packets filtered because the memo and/or the receiver were too big")
```

Comment:
```suggestion
                .with_description("Number of ICS-20 packets filtered because the memo and/or the receiver were exceeding the configured limits")
```